src/Dialect/Mlir/DialectBuilder.cpp

```diff
@@ -1641,11 +1631,6 @@ Value LLVMBuilder::null(Type type) const {
   return b().create<LLVM::NullOp>(loc(), type);
 }
 
-Value LLVMBuilder::nullI8Ptr() const {
-  Type I8PtrTy = LLVM::LLVMPointerType::get(b().getI8Type());
-  return b().create<LLVM::NullOp>(loc(), I8PtrTy);
-}
-
```

Comment:
Remove this since LLVMBuilder have no idea on switching between opaque and typed pointers.